<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="log_in" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>931</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>361</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Войдите, чтобы использовать систему</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>7</number>
        </property>
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="login">
          <property name="placeholderText">
           <string>логин</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <property name="bottomMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QLineEdit" name="pasword">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="placeholderText">
           <string>пароль</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="entry">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Войти</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="registr">
          <property name="text">
           <string>Зарегистрироваться</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Segoe Print&quot;;</string>
     </property>
     <property name="text">
      <string>IronTrans</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="registration1" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>931</width>
      <height>561</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>361</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Times New Roman&quot;;
margin-bottom:40px</string>
        </property>
        <property name="text">
         <string>Регистрация</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Укажите ваш статус:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <item>
         <widget class="QPushButton" name="as_guest">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Гость</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="as_worker">
          <property name="text">
           <string>Работник</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="registration_guest" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>931</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>361</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Times New Roman&quot;;
margin-bottom:40px</string>
        </property>
        <property name="text">
         <string>Регистрация</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="styleSheet">
             <string notr="true">padding:5px</string>
            </property>
            <property name="text">
             <string>Логин:     </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="login_guest">
            <property name="styleSheet">
             <string notr="true">margin-right:40px</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="styleSheet">
             <string notr="true">padding:5px</string>
            </property>
            <property name="text">
             <string>Пароль    </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pas_guest">
            <property name="styleSheet">
             <string notr="true">margin-right:40px</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="styleSheet">
           <string notr="true">padding:5px</string>
          </property>
          <property name="text">
           <string>Повторите 
пароль:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="again_guest">
          <property name="styleSheet">
           <string notr="true">margin-right:40px</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="signup_guest">
        <property name="styleSheet">
         <string notr="true">margin-left:50px;
margin-right:50px;
height:30%;</string>
        </property>
        <property name="text">
         <string>Зарегистрироваться</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Segoe Print&quot;;</string>
     </property>
     <property name="text">
      <string>IronTrans</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="registration_pro" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>124</y>
       <width>411</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Times New Roman&quot;;
margin-bottom:40px</string>
        </property>
        <property name="text">
         <string>Регистрация</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="styleSheet">
             <string notr="true">padding:5px</string>
            </property>
            <property name="text">
             <string>Логин:        </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="login_pro">
            <property name="styleSheet">
             <string notr="true">margin-right:40px</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">padding:5px</string>
            </property>
            <property name="text">
             <string>Должность:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="position_pro">
            <property name="styleSheet">
             <string notr="true">margin-right:40px</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="styleSheet">
             <string notr="true">padding:5px</string>
            </property>
            <property name="text">
             <string>Пароль       </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pas_pro">
            <property name="styleSheet">
             <string notr="true">margin-right:40px</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="styleSheet">
           <string notr="true">padding:5px</string>
          </property>
          <property name="text">
           <string>Повторите 
пароль:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="again_pro">
          <property name="styleSheet">
           <string notr="true">margin-right:40px</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="signup_pro">
        <property name="styleSheet">
         <string notr="true">margin-left:50px;
margin-right:50px;
height:30%;</string>
        </property>
        <property name="text">
         <string>Зарегистрироваться</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Segoe Print&quot;;</string>
     </property>
     <property name="text">
      <string>IronTrans</string>
     </property>
    </widget>
   </widget>
   <zorder>registration1</zorder>
   <zorder>registration_pro</zorder>
   <zorder>registration_guest</zorder>
   <zorder>log_in</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
